src/RpcServer/RpcServer.Wallet.cs

```diff
@@ -327,14 +327,14 @@ protected virtual JToken CancelTransaction(JArray _params)
             TransactionState state = NativeContract.Ledger.GetTransactionState(system.StoreView, txid);
             if (state != null)
             {
-                throw new RpcException(32700, "This tx is already confirmed, can't be cancelled.");
+                throw new RpcException(RpcErrorFactory.BadRequest("This tx is already confirmed, can't be cancelled."));
             }
 
             var conflict = new TransactionAttribute[] { new Conflicts() { Hash = txid } };
             Signer[] signers = _params.Count >= 2 ? ((JArray)_params[1]).Select(j => new Signer() { Account = AddressToScriptHash(j.AsString(), system.Settings.AddressVersion), Scopes = WitnessScope.None }).ToArray() : Array.Empty<Signer>();
             if (!signers.Any())
             {
-                throw new RpcException(32701, "no signers");
+                throw new RpcException(RpcErrorFactory.BadRequest("No signer"));
```

Comment:
And here we can use some generic `InternalServer` error with detailed `data` included. What do you think?

Reply:
Or InvalidParams

Reply:
Can be used as far.